CHANGELOG.md

```diff
@@ -1,39 +1,44 @@
-##2016-02-04 - Release 2.2.0
+## 2016-05-08 Release 2.2.0
 
-  - Deprecate the downloadURL parameter
-  - Add a systemd unit file for the RHEL 7 family
-  - Use a service_file_template variable to reference init scripts
-  - Add a context_path parameter to manage situations where Confluence is
-    served from a sub-path to the base URL
-  - Add manage_user parameter
-  - Fix minimum required stdlib version
+* Deprecate the downloadURL parameter
+* Add a systemd unit file for the RHEL 7 family
+* Use a service_file_template variable to reference init scripts
+* Add a context_path parameter to manage situations where Confluence is served from a sub-path to the base URL
+* Add manage_user parameter
+* Fix minimum required stdlib version
+* modulesync with voxpupuli defaults
 
-##2014-03-22 - Release 2.1.1
 
-  - Rewrite README file
-  - Bump confluence version to 5.7.1
-  - Update metadata, CHANGELOG to point to new namespace.
-  - Add .pmtignore file
+## 2014-03-22 Release 2.1.1
 
-##2014-03-22 - Release 2.1.0
+* Rewrite README file
+* Bump confluence version to 5.7.1
+* Update metadata, CHANGELOG to point to new namespace.
+* Add .pmtignore file
+
+
+## 2014-03-22 Release 2.1.0
 
 Note: This is the final release of this module before it is deprecated with a 999.999.999 version. This module will be moving the the puppet-community namespace on github and the puppet namespace on puppetforge soon.
```

Comment:
We should add the previous name, so we are historically accurate but it does not look like a zombie module.